assets/js/components/Work/CardItem.jsx

```diff
@@ -33,11 +30,13 @@ const WorkCardItem = ({
         </figure>
       </div>
       <div className="card-content">
-        <h3
-          className="title is-size-4"
-          dangerouslySetInnerHTML={{ __html: title ? title : "Untitled" }}
-        ></h3>
-
+        <h2
+          className="subtitle"
+          dangerouslySetInnerHTML={{
+            __html: title ? title : "Untitled",
+          }}
+        ></h2>
```

Comment:
@katdivyareddy10 Just curious why we're doing the `dangerouslySetInnerHTML` here?

Reply:
This is existing code.

Reply:
Got it, never mind:)